nss-sapphire-elephants/BangazonSite
# Setup

# Description

Setup database for sprint with Entity Framework

## Type of change

# Testing Instructions for Change Made

```git fetch --all```
```git checkout initial-setup```
Pull down and make sure data is good to go!

# Checklist:
